hat/app/org/hatdex/hat/she/mappers/DataEndpointMapper.scala

```diff
@@ -110,10 +110,10 @@ trait DataEndpointMapper extends JodaWrites with JodaReads {
           s"- ${t.minusMinutes(1).toString("dd MMMM")}"
         }
         else {
-          s"- ${t.withZone(start.getZone).toString("dd MMMM HH:mm z")}"
+          s"- ${t.withZone(start.getZone).toString("dd MMMM HH:mm ZZZ")}"
```

Comment:
Why the change in format? I wonder how the front end will react to that

Reply:
This was due to undocumented behaviour in the Joda library causing the tests to fail. I was unable to find a more satisfactory solution that would have the same output.